how-to/debug-buildfarm-builder.rst

```diff
@@ -26,6 +32,35 @@ To anoint yourself to be part of that team in qastaging, see
 
 3. Click the "Switch to manual-mode" button.
 
+.. note::
+
+   Setting the builder to ``manual`` prevents new builds to be dispatched to
```

Comment:
s/manual/disabled

Reply:
I removed the section as agreed with @ineesalmeida